src/main/resources/logback.xml

```diff
@@ -0,0 +1,43 @@
+<configuration>
+    <appender name="JSON-STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <target>System.out</target>
+        <encoder class="net.logstash.logback.encoder.LoggingEventCompositeJsonEncoder">
+            <providers>
+                <message/>
```

Comment:
It would be nice if we have the following order:

```
1. <timestamp>
2. <logLevel>
3. <loggerName>
4. <message/>
5. <stackTrace>
6. <arguments/>
```

Reply:
is order an important thing for a JSON document?

Reply:
In general, the order doesn't matter on JSON. But in this case, this is simply passed as an event object and when it prints out by the console print or stackdriver printer, it just prints the same order that JSON holds. So, If I run sync-java in GCP, it would be nice to see the order which I mentioned. Because typically they are showing a trimmed JSON.

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/260/commits/ebcae773e2111a431a1fdd8ea36b3930292f7d68

Reply:
Sample output: 
````
{"timestamp":"2021-04-09T09:18:12.461+02:00","severity":"ERROR","loggerName":"com.commercetools.project.sync.CliRunner","message":"Failed to run sync process.","stackHash":"b3d4f391","stackTrace":"c.c.p.s.e.CliException: Unknown argument \"ahmet\" supplied to \"-s\" or \"--sync\" option! Choose one or more of the following modules to run: \"types\", \"productTypes\", \"cartDiscounts\", \"customObjects\", \"categories\", \"inventoryEntries\", \"states\", \"taxCategories\", \"customers\", \"products\" or \"shoppingLists\".\n"}

````